### ISSUE TITLE
'snapToPointer' parameter for Draw interaction?

### ISSUE THREAD

**Issue by edellucien** [openlayers/openlayers#13798]
**Is your feature request related to a problem? Please describe.**
I think we have the same problem when working with Snap & Draw interaction as it used to be with Modify & Snap on this previous issue:
https://github.com/openlayers/openlayers/issues/11976
When trying to Draw feature we need the ability to snap to a layer that is actually not the Draw interaction's source.

**Describe the solution you'd like**
Could we solve this with the same parameter 'snapToPointer' as for Modify?
Similar to this but for Draw interaction:
https://github.com/openlayers/openlayers/pull/11978/files

Thanks 😄

**Comment by ahocevar** [comment]
You can configure the `Snap` interaction with a source, and it does not have to be the same sources that your `Draw` interaction uses. Does that help?

**Comment by edellucien** [comment]
You are right the Draw interaction is correct. I had a problem with a Pointer interaction which was handling several sources simultameously, which I replaced with multiple Modify interactions, with the option 'snapToPointer'.
It works fine now.

Thanks for the fast answer and sorry that I bothered you.
Have a nice day 😄 👍
